samanthas_telegram_bot/api_queries/check.py

```diff
@@ -0,0 +1,47 @@
+"""Functions for checking existence of entities with the backend."""
+import logging
+
+import httpx
+
+from samanthas_telegram_bot.data_structures.constants import API_URL_PREFIX
+
+logger = logging.getLogger(__name__)
+# TODO check for something different in case host is unavailable? Add decorators to all functions?
+#  httpx.ConnectTimeout
+
+
+async def chat_id_is_registered(chat_id: int) -> bool:
+    """Checks whether the chat ID is already stored in the database."""
+    logger.info(f"Checking with the backend if chat ID {chat_id} exists...")
+
+    async with httpx.AsyncClient() as client:
+        r = await client.get(
+            f"{API_URL_PREFIX}/personal_info/check_existence_of_chat_id/",
+            params={"registration_telegram_bot_chat_id": chat_id},
+        )
+    if r.status_code == httpx.codes.OK:
+        logger.info(f"... {chat_id} already exists")
```

Comment:
All of this looks like debug logging.

Reply:
Maybe, but I still think it's not low-level enough. Like from `httpx` documentation, for example:
```
DEBUG [2023-03-16 14:36:21] httpcore - http11.receive_response_headers.complete return_value=(b'HTTP/1.1', 200, b'OK', [(b'Content-Encoding', b'gzip'), (b'Accept-Ranges', b'bytes'), (b'Age', b'507675'), (b'Cache-Control', b'max-age=604800'), (b'Content-Type', b'text/html; charset=UTF-8'), (b'Date', b'Thu, 16 Mar 2023 14:36:21 GMT'), (b'Etag', b'"3147526947+ident"'), (b'Expires', b'Thu, 23 Mar 2023 14:36:21 GMT'), (b'Last-Modified', b'Thu, 17 Oct 2019 07:18:26 GMT'), (b'Server', b'ECS (nyb/1D2E)'), (b'Vary', b'Accept-Encoding'), (b'X-Cache', b'HIT'), (b'Content-Length', b'648')])
INFO [2023-03-16 14:36:21] httpx - HTTP Request: GET https://www.example.com "HTTP/1.1 200 OK"
DEBUG [2023-03-16 14:36:21] httpcore - http11.receive_response_body.started request=<Request [b'GET']
```
So even a GET request is an INFO for them.

But I will think about it later - I'll definitely leave user created in INFO, and the rest will depend on what I want to pay special attention to in the workflow: there I would I want the log messages displayed as INFO despite them being more like DEBUG.